Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/WindowsWebAppZipDeployProvider.ts

```diff
@@ -3,6 +3,7 @@ import tl = require('azure-pipelines-task-lib/task');
 import { FileTransformsUtility } from '../operations/FileTransformsUtility';
 import * as ParameterParser from 'azure-pipelines-tasks-webdeployment-common-v4/ParameterParserUtility';
 import { DeploymentType } from '../operations/TaskParameters';
+import { addReleaseAnnotation } from '../operations/ReleaseAnnotationUtility';
```

Comment:
Instead of adding a function directly, can we consider to create an interface for it? As it is like in other imports (e.g. `FileTransformsUtility `). What do you think?

Reply:
As we already have discussed in the meeting, when you push your commit related to moving logic to parent method, we can mark as solved. Thanks for your effort!

Reply:
So, do we still need this import?